### PR TITLE
add facebook pixel

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

adds a tracking pixel that reports to Facebook that you've read something on the guardian (if you're logged in to Facebook)
## What is the value of this and can you measure success?

it should mean our ad-spend on Facebook can be better targeted 
## Does this affect other platforms - Amp, Apps, etc?

AMP
## Request for comment

@stephanfowler 

tell facebook that you've read the guardian

### CODE REVIEW

**Comment by JustinPinner** [comment]
😭

**Comment by sndrs** [comment]
@NataliaLKB is the amp bit ok?

File: common/app/views/mainAMP.scala.html [hunk 1]
```diff
@@ -36,6 +37,7 @@
                 <amp-pixel src="//beacon.guim.co.uk/count/pva.gif"></amp-pixel>
             }
         }
+        <amp-pixel src="//www.facebook.com/tr?id=@FBPixel.account&ev=PageView&noscript=1"></amp-pixel>
```
Comment (stephanfowler) [inline common/app/views/mainAMP.scala.html]: Is there any harm in just fixing the protocol to https?

**Comment by stephanfowler** [comment]
👍 - quite apart from the emotional cost

File: common/app/views/mainAMP.scala.html (hunk 1 above)
Comment (sndrs) [inline common/app/views/mainAMP.scala.html]: no idea, was just following what was there – is it a req @NataliaLKB?

**Comment by sndrs** [comment]
66_cctv_155_big

**Comment by NataliaLKB** [comment]
Amp bit looks fine 👍  The validator says it should be okay too

File: common/app/views/mainAMP.scala.html (hunk 1 above)
Comment (NataliaLKB) [inline common/app/views/mainAMP.scala.html]: I think either is fine

File: common/app/views/mainAMP.scala.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 @import common.{AnalyticsHost, CanonicalLink, LinkTo}
 @import conf.Configuration
 @import views.support.OmnitureAnalyticsData
+@import views.support.FBPixel
```
Comment (NataliaLKB) [inline common/app/views/mainAMP.scala.html]: Does it make more sense to write this as:
`@import views.support.{OmnitureAnalyticsData, FBPixel}`

Comment (NataliaLKB) [inline common/app/views/mainAMP.scala.html]: Bah sorry! I was a little late.

**Comment by paperboyo** [comment]
Nationalise them already! 😡

maxresdefault

**Pull request merged**